rtd-requirements.txt

```diff
@@ -0,0 +1,7 @@
+# Need to install the api/sdk in the venv for autodoc. Used by .readthedocs.yml
+opentelemetry-api @ git+https://github.com/open-telemetry/opentelemetry-python.git@main#egg=opentelemetry-api&subdirectory=opentelemetry-api
```

Comment:
I'm confused how this is supposed to work on release branches?

Reply:
Are we publishing docs from the releases branch?  

The idea here is to separate RTD requirements from tox Docs CI requirements. In CI, we run `tox—e docs`, which will install core packages using the `CORE_REPO_SHA` environment variable. We usually update the SHA during releases to point to the respective release branch in the core, right? So, this will make Docs CI pass in contrib.

Reply:
uhm maybe this is a thing only in core, sorry for the noise